cmd/flags.go

```diff
@@ -1,35 +1,40 @@
 package cmd
 
 import (
+	"crypto/tls"
+	"crypto/x509"
 	"flag"
+	"io/ioutil"
 	"net/http"
 	_ "net/http/pprof"
+	"os"
+	"time"
 
 	"github.com/getlantern/goexpr/isp"
 	"github.com/getlantern/goexpr/isp/ip2location"
 	"github.com/getlantern/goexpr/isp/maxmind"
 	"github.com/getlantern/golog"
-	tlsredis "github.com/getlantern/tlsredis"
-	"gopkg.in/redis.v5"
+
 	"strings"
+
+	rclient "github.com/go-redis/redis"
 )
 
 var (
 	log = golog.LoggerFor("cmd")
 )
 
 var (
-	Schema          = flag.String("schema", "schema.yaml", "Location of schema file, defaults to ./schema.yaml")
-	AliasesFile     = flag.String("aliases", "", "Optionally specify the path to a file containing expression aliases in the form alias=template(%v,%v) with one alias per line")
-	EnableGeo       = flag.Bool("enablegeo", false, "enable geolocation functions")
-	ISPFormat       = flag.String("ispformat", "ip2location", "ip2location or maxmind")
-	ISPDB           = flag.String("ispdb", "", "In order to enable ISP functions, point this to a ISP database file, either in IP2Location Lite format or MaxMind GeoIP2 ISP format")
-	RedisAddr       = flag.String("redis", "", "Redis address in \"redis[s]://host:port\" format")
-	RedisCA         = flag.String("redisca", "", "Certificate for redislabs's CA")
-	RedisClientPK   = flag.String("redisclientpk", "", "Private key for authenticating client to redis's stunnel")
-	RedisClientCert = flag.String("redisclientcert", "", "Certificate for authenticating client to redis's stunnel")
-	RedisCacheSize  = flag.Int("rediscachesize", 25000, "Configures the maximum size of redis caches for HGET operations, defaults to 25,000 per hash")
-	PprofAddr       = flag.String("pprofaddr", "localhost:4000", "if specified, will listen for pprof connections at the specified tcp address")
+	Schema         = flag.String("schema", "schema.yaml", "Location of schema file, defaults to ./schema.yaml")
+	AliasesFile    = flag.String("aliases", "", "Optionally specify the path to a file containing expression aliases in the form alias=template(%v,%v) with one alias per line")
+	EnableGeo      = flag.Bool("enablegeo", false, "enable geolocation functions")
+	ISPFormat      = flag.String("ispformat", "ip2location", "ip2location or maxmind")
+	ISPDB          = flag.String("ispdb", "", "In order to enable ISP functions, point this to a ISP database file, either in IP2Location Lite format or MaxMind GeoIP2 ISP format")
+	redisAddr      = flag.String("redis", "", "Redis address in \"redis[s]://host:port\" format")
```

Comment:
Is it on purpose that these become private now?

Reply:
Also, are we no longer authenticating the client with a client cert?

Reply:
yeah there's no client auth on IBM (or most providers I don't think), but they're private just because there was no reason for them to be public, or at least none I could see.

Reply:
Yeah, I believe borda used to reference them but now it has its own copies of them, so it should be okay to make these non-exported.

Reply:
One side note about not using client auth anymore. IIRC we instituted that because we considered that more secure than just a REDIS AUTH password by itself. Not sure if we have any concerns about that now ...

Reply:
Yeah no question having client auth adds a degree of security vs no client auth, but we just don't have that option here or with any cloud provider I don't believe. IBM has another layer of protection in that the domain itself appears to be securely random, so as long as the bad guys don't figure out that and the password we're good.